marketplace-service/src/main/java/com/axonivy/market/github/util/GitHubUtils.java

```diff
@@ -132,4 +148,99 @@ public static InputStream extractedContentStream(GHContent content) {
       return null;
     }
   }
+
+  public static Dependabot getDependabotAlerts(GHRepository repo, GHOrganization organization, String accessToken) {
+    Dependabot dependabot = new Dependabot();
+    try {
+      ResponseEntity<List<Map<String, Object>>> response = fetchApiResponseAsList(accessToken,
+          String.format(GitHubConstants.Url.REPO_DEPENDABOT_ALERTS_OPEN, organization.getLogin(), repo.getName()));
+      dependabot.setStatus(com.axonivy.market.enums.AccessLevel.ENABLED);
```

Comment:
Add import static for enum ENABLED instead of using full path of enum class in a method.
Update also for DISABLED and NO_PERMISSION.